tracer/src/Datadog.Trace/AppSec/Security.cs

```diff
@@ -442,6 +442,15 @@ private void InitWafAndInstrumentations(bool fromRemoteConfig = false)
                 {
                     _configurationStatus.FallbackEmbeddedRuleSet ??= RuleSet.From(_wafInitResult.EmbeddedRules);
                 }
+
+                if (!fromRemoteConfig)
+                {
+                    // occurs the first time we have initialize the WAF
+                    TelemetryFactory.Metrics.SetWafVersion(_waf!.Version);
+                    TelemetryFactory.Metrics.RecordCountWafInit();
```

Comment:
dont we want to o`WafUpdates` one as well? in the [RFC ](https://docs.google.com/document/d/17dYHulDMa9XlZFWBPd4bAdFF-V1fU_0CYDLLMzcgilE/edit#heading=h.jq39arln8xio) it says "This metric (waf.updates) is emitted when rules are updated (or reloaded) by the WAF (after a rule update coming from remote configuration for instance)"